jira.go

```diff
@@ -250,6 +250,13 @@ func openJiraTickets(flags flags, projectInfo jsn.Json, vulnsForJira map[string]
 				fullListNotCreatedIssue += displayErrorForIssue(vulnForJira, "ifUpgradeAvailableOnly", errors.New(message), "", customDebug)
 				continue
 			}
+		} else if flags.optionalFlags.canAutoPR {
+			// skip ticket creating if the vuln is not fixable and not upgradable
+			if !(jsonVuln.K("fixInfo").K("isFixable").Bool().Value || jsonVuln.K("fixInfo").K("isUpgradable").Bool().Value) {
```

Comment:
Curious why both checks are needed? ```jsonVuln.K("fixInfo").K("isFixable").Bool().Value || jsonVuln.K("fixInfo").K("isUpgradable").Bool().Value)```
`isFixable` should be enough as it checks it is fixable by Snyk aka we can raise a PR for upgrade or a patch

Reply:
@lili2311 that's true. I was working off of the parameters set out in the linked issue which requested that if either `isUpgradable` or `isFixable` are true. I can change the logic to just check `isFixable` if you agree

Reply:
@lili2311 I have made changes to only check against `isFixable`, let me known what you think and if there is anything else